### PR TITLE
OV separated exec

### PR DESCRIPTION
**Pull request by pfinashx**
Support paralleling start OpenVino part of End-2-End test

### CODE REVIEW

**Comment by FyodorK** [comment]
Jenkins please retry a build

**Comment by FyodorK** [comment]
Jenkins please retry a build

**Comment by FyodorK** [comment]
Jenkins please retry a build

**Comment by FyodorK** [comment]
Jenkins please retry a build

**Comment by pfinashx** [comment]
@vshampor Please take a look

File: tests/test_sota_checkpoints.py [hunk 1]
```diff
@@ -466,46 +416,22 @@ def test_openvino_eval(self, openvino, sota_checkpoints_dir, sota_data_dir, ov_d
...
                 exit_code, err_str = self.run_cmd(ac_cmd, cwd=ACC_CHECK_DIR)
-
-        ac_metric = self.read_csv_metric(eval_test_struct.model_name_)
-        fp32_ac_metric = None
-        if eval_test_struct.reference_ is not None:
-            if os.path.exists('{}.csv'.format(eval_test_struct.reference_)):
-                fp32_ac_metric = self.read_csv_metric(eval_test_struct.reference_)
+                if exit_code != 0 or err_str is not None:
+                    assert False
+                assert True
```
Comment (vshampor) [inline tests/test_sota_checkpoints.py:421]: `assert True` is redundant

File: tests/test_sota_checkpoints.py [hunk 2]
```diff
@@ -466,46 +416,22 @@ def test_openvino_eval(self, openvino, sota_checkpoints_dir, sota_data_dir, ov_d
...
                                                   PROJECT_ROOT, config=eval_test_struct.model_name_)
                 exit_code, err_str = self.run_cmd(ac_cmd, cwd=ACC_CHECK_DIR)
-
-        ac_metric = self.read_csv_metric(eval_test_struct.model_name_)
-        fp32_ac_metric = None
-        if eval_test_struct.reference_ is not None:
-            if os.path.exists('{}.csv'.format(eval_test_struct.reference_)):
-                fp32_ac_metric = self.read_csv_metric(eval_test_struct.reference_)
+                if exit_code != 0 or err_str is not None:
+                    assert False
```
Comment (vshampor) [inline tests/test_sota_checkpoints.py:420]: Use `pytest.fail` instead to fail with an explicit message.

File: tests/test_sota_checkpoints.py [hunk 3]
```diff
@@ -466,46 +416,22 @@ def test_openvino_eval(self, openvino, sota_checkpoints_dir, sota_data_dir, ov_d
...
+                assert True
             else:
-                fp32_ac_metric = self.ref_fp32_dict[str(eval_test_struct.reference_)]
-
-        if ac_metric:
-            diff_target = round((ac_metric - eval_test_struct.expected_), 2)
-            diff_fp32 = round((ac_metric - fp32_ac_metric), 2) if fp32_ac_metric is not None else None
+                assert False
```
Comment (vshampor) [inline tests/test_sota_checkpoints.py:423]: Same here and everywhere

File: tests/test_sota_checkpoints.py [hunk 4]
```diff
@@ -466,46 +416,22 @@ def test_openvino_eval(self, openvino, sota_checkpoints_dir, sota_data_dir, ov_d
...
-                                      diff_target,
-                                      eval_test_struct.diff_fp32_min_,
-                                      eval_test_struct.diff_fp32_max_,
-                                      fp32_ac_metric,
-                                      diff_fp32,
-                                      eval_test_struct.diff_target_min_,
-                                      eval_test_struct.diff_target_max_)
-
-        self.color_dict[eval_test_struct.model_name_], is_accuracy_within_thresholds = retval
-        assert is_accuracy_within_thresholds
```
Comment (vshampor) [inline tests/test_sota_checkpoints.py:500]: Where did all this code go? Where is the logic that checks the metrics against the thresholds now?

**Review by vshampor — CHANGES_REQUESTED** [review]

File: tests/test_sota_checkpoints.py (hunk 4 above)
Comment (pfinashx) [reply tests/test_sota_checkpoints.py:500]: The logic that checks thresholds for OpenVino is postponed in separate script in Jenkins trigger job.

File: tests/test_sota_checkpoints.py (hunk 2 above)
Comment (pfinashx) [reply tests/test_sota_checkpoints.py]: Done

File: tests/test_sota_checkpoints.py (hunk 3 above)
Comment (pfinashx) [reply tests/test_sota_checkpoints.py]: Done

File: tests/test_sota_checkpoints.py (hunk 1 above)
Comment (pfinashx) [reply tests/test_sota_checkpoints.py]: Removed

**Comment by pfinashx** [comment]
Jenkins please retry a build

**Review by vshampor — APPROVED** [review]

**Comment by vshampor** [comment]
Jenkins please retry a build

**Comment by vshampor** [comment]
Jenkins please retry a build

**Comment by FyodorK** [comment]
Jenkins please retry a build

**Pull request merged**